src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1032,10 +1042,29 @@ namespace winrt::TerminalApp::implementation
                 this->_SplitPane(SplitDirection::Automatic,
                                  0.5f,
                                  newPane);
+
+                if (auto autoPeer = Automation::Peers::FrameworkElementAutomationPeer::FromElement(*this))
+                {
+                    autoPeer.RaiseNotificationEvent(
+                        Automation::Peers::AutomationNotificationKind::ActionCompleted,
+                        Automation::Peers::AutomationNotificationProcessing::ImportantMostRecent,
+                        fmt::format(std::wstring_view{ RS_(L"SplitPaneAnnouncement") }, _settings.GetProfileForArgs(newTerminalArgs).Name()),
+                        L"NewSplitPane" /* unique name for this notification category */);
+                }
             }
             else
             {
                 _CreateNewTabFromPane(newPane);
+                if (auto autoPeer = Automation::Peers::FrameworkElementAutomationPeer::FromElement(*this))
+                {
+                    // we can't check if this is a leaf pane,
+                    // but getting the profile returns null if we aren't, so that works!
+                    autoPeer.RaiseNotificationEvent(
+                        Automation::Peers::AutomationNotificationKind::ActionCompleted,
+                        Automation::Peers::AutomationNotificationProcessing::ImportantMostRecent,
+                        fmt::format(std::wstring_view{ RS_(L"NewTabAnnouncement") }, _settings.GetProfileForArgs(newTerminalArgs).Name()),
+                        L"NewTab" /* unique name for this notification category */);
+                }
```

Comment:
I could have sworn that we unified all of the places tabs and panes could be created... Did we not? Why do we have to put these hooks in four or five places?

Reply:
Because of commandline args hahaha. The first commit took advantage of that unification: https://github.com/microsoft/terminal/pull/13575/commits/2cb1ff753fef7276cd36330438ea8cc8fc732342

But then we decided we didn't want that behavior for commandline args.